src/styles/sidebar/components/annotation.scss

```diff
@@ -17,24 +17,6 @@
   .annotation-action-btn {
     color: var.$grey-6;
   }
-
-  .annotation-header__timestamp-link {
-    color: var.$grey-5;
-  }
-}
-
-.annotation {
-  display: block;
-  font-family: var.$sans-font-family;
-  position: relative;
-}
-
-.annotation.is-dimmed {
```

Comment:
I believe this selector is still referenced in a not-very-discoverable way here:

https://github.com/hypothesis/client/blob/23f130275bb1fbb624d9dc7d376401885e227091/src/sidebar/components/annotation-thread.js#L52